lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,17 @@
+<%page expression_filter="h" args="notification_type, notification_icon, submission_response_message, points_earned, points_possible"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "partially-correct"
+    notification_icon = "asterisk"
+    submission_response_message = "Partially correct"
+    points_earned = "3"
+    points_possible = "4"
+%>
+
+<div class="submission_response ${notification_type}">
```

Comment:
Do you want me to provide comments like this?
Usage                     notification_type      notification_icon
Hints                       info                            question
Partially correct     partially-correct       asterisk